2-planning/0-introduction.adoc

```diff
@@ -0,0 +1,56 @@
+= Introduction
+
+When I was younger and dumber, the planning stage for APIs was pretty much
+"think of a list of endpoints then start coding". This is wildly insufficient.
+As the <<theory-contracts,API Contracts>> chapter explained, contracts are
+incredibly important throughout the life-cycle of an API. We are going to look
+into how they can be used in the planning and design stage.
+
+Most of the planning and design phase is intended to do two things:
+
+1. Make developers think really hard about the requirements for their API 2.
+Gather feedback from clients application developers 3. Use the feedback to
```

Comment:
`2.` and `3.` should be on separate lines…

Reply:
Ha thats my auto-wrap extension eatingn lines, thank you.